src/schedule/ucc_schedule.h

```diff
@@ -104,6 +104,7 @@ typedef struct ucc_coll_task {
     double   start_time; /* timestamp of the start time:
                             either post or triggered_post */
     uint32_t seq_num;
+    int is_service; /* bool flag specifying if using a special service worker */
```

Comment:
this is smth specific to TL/UCP - should not be in common core data structure imho.

Reply:
It has been removed, now `is_service` is deduced from the team